test/unit/util/fsTest.ts

```diff
@@ -5,6 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { expect } from 'chai';
+import * as fsLib from 'fs';
```

Comment:
Shouldn't this be` import * as fsLib from 'src/util/fs'`?

Reply:
I could add `statSync` to `util/fs`